authentik/stages/email/templates/email/password_reset.txt

```diff
@@ -5,9 +5,9 @@ You recently requested to change your password for your authentik account. Use t
 {% endblocktrans %}
 {{ url }}
 {% blocktrans with expires=expires|naturaltime %}
-If you did not request a password change, please ignore this Email. The link above is valid for {{ expires }}.
+If you did not request a password change, please ignore this email. The link above is valid for {{ expires }}.
 {% endblocktrans %}
 
--- 
+--
```

Comment:
```suggestion
-- 
```

This trailing space should stay. Cf. RFC 3676 sec. 4.3

Reply:
I am blown away. Thank you.